gen/build/src/lib.rs

```diff
@@ -425,8 +425,9 @@ fn best_effort_copy_headers(src: &Path, dst: &Path, max_depth: usize) {
             }
             Ok(file_type) if file_type.is_file() => {
                 let src = entry.path();
-                if src.extension() != Some(OsStr::new("h")) {
-                    continue;
+                match src.extension().map(|x| x.to_str()).flatten() {
```

Comment:
FYI a more compact way to write this is:

```suggestion
                match src.extension().and_then(OsStr::to_str) {
```